extern/bullet2/src/BulletCollision/CollisionShapes/btConeShape.h

```diff
@@ -43,6 +43,15 @@ ATTRIBUTE_ALIGNED16(class) btConeShape : public btConvexInternalShape
 	btScalar getRadius() const { return m_radius;}
 	btScalar getHeight() const { return m_height;}
 
+	virtual void setRadius(btScalar radius)
```

Comment:
that change was upstream to bullet sources already and it is included in new bullet 2.85